### PR TITLE
atomic creation

### PR DESCRIPTION
**Pull request by motu55**

### CODE REVIEW

**Comment by pascalfree** [comment]
Shouldn't the new_signup function be atomic? In case it gets called from elsewere.

**Comment by motu55** [comment]
yes and no
yes if we make the method transactional any place it is called it is "safe"
no
if the request fails for other reasons (a method next to it (ok at the moment there are few but there might be more)) the user will try to reload in anticipation something else will happen, therefore the method will be called again which tries to add the "new" member which fails because it was succesfull the first time (but something else failed) and will from then always throw an 500. So oit makes sense to make whole requests atomic.

Basically all requests which update more than one object should be atomic

**Review by pascalfree — APPROVED** [review]
